cmds/tui.c

```diff
@@ -1273,12 +1293,13 @@ static void print_graph_indent(struct tui_graph *graph, struct tui_graph_node *n
 			continue;
 		}
 
-		if (i < depth - 1 || single_child)
-			printw("  │");
+		if (i < depth - 1 || single_child) {
+			printw("  %s", vertical_line);
+		}
```

Comment:
You don't need the curly brackets for a statement.

Reply:
Thanks for review.